application/services/telegram.py

```diff
@@ -54,31 +70,31 @@ def send_detection_message(camera, timestamp, feed_url, img_filename):
 
 def request(method, endpoint, params, files):
     """
-    Function to make requests to Telegram.
+    Function to make requests to Telegram's bot API.
 
     params:
         method: 'post' or 'get' HTTP methods
-        endpoint: Any Telegram API endpoint. i.e 'sendMessage'
+        endpoint: Any Telegram bot API endpoint. i.e 'sendMessage'
         params: dict of url params
         files: dict of files
     """
     bot_token = vault.get_value("EXTERNAL_API", "telegram", "token")
+    api_url = vault.get_value("external_api", "telegram", "url")
```

Comment:
Let's capitalize this to stay consistent :D

Reply:
💯 😃